src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  setTodo: (id: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, setTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(todo.userId).then(setUser);
+  }, []);
```

Comment:
Fix it 
<img width="937" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/64750958/b85182a8-c270-4580-800d-484afb93f72e">